README.md

```diff
@@ -1,6 +1,12 @@
 # ActiveService [![Build Status](https://travis-ci.org/bilby91/active_service.svg?branch=master)](https://travis-ci.org/bilby91/active_service) [![Code Climate](https://codeclimate.com/github/bilby91/active_service/badges/gpa.svg)](https://codeclimate.com/github/bilby91/active_service)
 
-Steroids for business clases :muscle: 
+Steroids for business clases :muscle:
+
+`ActiveService` was created with the purpose of moving logic that usually live in your controller to a separate layer in charge of business logic, leaving the parameters handling, format type responses (JSON, XML, HTML, etc) and other stuff to the controller.
```

Comment:
`ActiveService` purpose is to move logic that usually lives in your controller to a separate layer in charge of business logic, leaving the parameters handling, format type responses (JSON, XML, HTML, etc) and other stuff to the controller.